packages/eas-cli/src/vcs/git.ts

```diff
@@ -127,6 +127,15 @@ export default class GitClient extends Client {
       !trackedFiles.includes(pathWithoutLeadingDot)
     );
   }
+
+  public async isFileIgnoredAsync(filePath: string): Promise<boolean> {
+    try {
+      const { status } = await spawnAsync('git', ['check-ignore', '-q', filePath]);
```

Comment:
if i remember correctly spawn async always throws if exit code is non zero

Reply:
You're right https://github.com/expo/spawn-async/blob/master/src/spawnAsync.ts#L70